lib/ecs_deploy/instance_fluctuation_manager.rb

```diff
@@ -50,11 +52,18 @@ def decrease
 
       container_instance_arns = ecs_client.list_container_instances(
         cluster: @cluster
-      ).container_instance_arns
-      container_instances = ecs_client.describe_container_instances(
-        cluster: @cluster,
-        container_instances: container_instance_arns
-      ).container_instances
+      ).flat_map(&:container_instance_arns)
+      container_instances = container_instance_arns.each_slice(MAX_DESCRIBABLE_ECS_CONTAINER_COUNT).flat_map do |arns|
```

Comment:
You can get `container_instances` without introducing `MAX_DESCRIBABLE_ECS_CONTAINER_COUNT` by writing like below:

```ruby
container_instances = ecs_client.list_container_instances(cluster: @cluster).flat_map do |resp|
  resp.container_instance_arns.flat_map do |arns|
    ecs_client.describe_container_instances(
      cluster: @cluster,
      container_instances: arns
    ).container_instances
  end
end
```